language/evm/stdlib/sources/U256.move

```diff
@@ -0,0 +1,18 @@
+#[evm_arith]
+module 0x1::U256 {
+    native struct U256 has copy, drop, store;
+    native public fun u256_from_words(lo: u128, hi: u128): U256;
+    native public fun add(x: U256, y: U256): U256;
```

Comment:
I'm wondering whether it would be possible to extend the frontend to allow for `x + y` etc., and be able to use u256 (small case) without import (like vector),  Then at the moment where the bytecode is generated we insert the call to the native function. The current way how we represent this doesn't seem to create a good UX compared to Solidity. @tnowacki @vgao1996